### ISSUE TITLE
bug in make_snp_file.py

### ISSUE THREAD

**Issue by weizhu365** [mhguo1/PBTest#15]
In the line 389 of the file make_snp_file.py, the function "test_include_vep" should be replaced with "test_exclude_vep" (see the snapshot below). Besides, the codes in the lines 387 and 388 also needs to be modified accordingly (use exclude not include). 

![image](https://user-images.githubusercontent.com/4519424/88716786-955dcb00-d0ed-11ea-8fd2-80bb00ffd2f7.png)

Obviously, the bug causes the option of --excludevep has no use.

Thanks,

Wei

**Comment by wanlee-yang** [comment]
You are right, I also found this problem recently when using this tool for rare variants burden test. if you wanted to use --excludevep, the includevep in line 387-389 should be changed to excludevep, otherwise an error will be reported.
